src/main/thrift/parquet.thrift

```diff
@@ -237,6 +237,38 @@ struct SizeStatistics {
    3: optional list<i64> definition_level_histogram;
 }
 
+/**
+ * Bounding box of geometries in the representation of min/max value pair of
+ * coordinates from each axis. Values of Z and M are omitted for 2D geometries.
+ */
+struct BoundingBox {
+  1: optional double x_min;
+  2: optional double x_max;
+  3: optional double y_min;
+  4: optional double y_max;
+  5: optional double z_min;
+  6: optional double z_max;
+  7: optional double m_min;
+  8: optional double m_max;
+}
+
+/** Statistics specific to GEOMETRY logical type */
+struct GeometryStatistics {
+  /** Bounding box of geometries */
+  1: optional BoundingBox bbox;
+  /** Covering of geometries as a list of Google S2 cell ids */
+  2: list<i64> s2_cell_ids;
+  /** Covering of geometries as a list of Uber H3 indices */
+  3: list<i64> h3_indices;
```

Comment:
Is it really forward-looking to hardcode company-specific statistics in the schema?

Reply:
I am not super familiar with thrift's compatibility, but is it possible to leave this out for now, and add additional fields later without breaking backcompat? 

If that's the case, I think I would prefer leaving those out for now.

Reply:
I agree that S2 and H3 are probably too new to add at this time. The main idea is to show we can extend more specific types of geospatial statistics at this place. 

BTW, S2 and H3 are widely used and licensed under Apache 2.0, we can remove the specific company names when we decide to add them.

Reply:
I am fine either way (adding s2/h3 or leave them out). But if we want to add them, don't we also need to provide the precision / level of these IDs? Or one can tell the precision from the cell id itself?

Reply:
I agree that I have missed precision / level of these IDs...